custom-cakes/modules/how-to-order/__step/_first/how-to-order__step_first.css

```diff
@@ -0,0 +1,3 @@
+.how-to-order__step_first::before {
```

Comment:
I can't agree with this approach. The number of steps may change from time to time and we can't afford to add a new css file for every number.

Reply:
fixed. I used spans with a common class (how-to-order__step-conter) instead of pseudo elements. All styles for this class are in one file now.